### PR TITLE
Enable a bunch of CSS tests

### PR DESCRIPTION
**Pull request by jdm**
These are all directories that contain tests for features that we implement, as well as tests for features that we don't.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/15550)

### CODE REVIEW

File: tests/wpt/metadata-css/compositing-1_dev/html/mix-blend-mode-creates-stacking-context.htm.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[mix-blend-mode-creates-stacking-context.htm]
+  type: testharness
+  [CSS Test: an element with mix-blend-mode other than normal creates a stacking context]
```
Comment (emilio) [inline tests/wpt/metadata-css/compositing-1_dev/html/mix-blend-mode-creates-stacking-context.htm.ini]: Cool, these will actually test stuff like https://github.com/servo/webrender/pull/868

**Review by emilio — APPROVED** [review]

**Comment by jdm** [comment]
@bors-servo: retry
* #15570

**Pull request merged**